scripts/cades_shpc_condo_cmake

```diff
@@ -11,12 +9,12 @@ ARGS=(
     -D CMAKE_BUILD_TYPE=Debug
     -D BUILD_SHARED_LIBS=ON
     ### COMPILERS AND FLAGS ###
-    -D CMAKE_CXX_FLAGS="-g -arch=${GPU_ARCH} -lineinfo \
```

Comment:
so now extended lambda only gets passed to cxx11 flags? What exactly does extended lambda do again?

Reply:
See  #353

Reply:
https://devblogs.nvidia.com/new-compiler-features-cuda-8/